app/server/db-connect.ts

```diff
@@ -0,0 +1,57 @@
+// lib/dbConnect.tsx
+
+import type _mongoose from 'mongoose';
+import { connect } from 'mongoose';
+
+declare global {
+  // eslint-disable-next-line
+  var mongoose: {
+    promise: ReturnType<typeof connect> | null;
+    conn: typeof _mongoose | null;
+  };
+}
+
+const { MONGODB_URI } = process.env;
```

Comment:
are you sure this is the best way of doing it? it might be a good idea to add a constants file, which makes life a lot easier